enumfields/enums.py

```diff
@@ -36,6 +36,9 @@ def __new__(mcs, name, bases, attrs):
                 if k != 'value':
                     setattr(m, k, v)
 
+        # uniqueness check - choices must be unique
+        unique(obj)
```

Comment:
It does not work for child classes when used the standard way (as decorator), so I am calling it here, in metaclass. The effect is the same and it works for child classes as well.

Reply:
good job

Reply:
you can use here 

`return unique(obj)`